ts/parts/Interaction.ts

```diff
@@ -52,7 +52,7 @@ declare global {
             importEvents(): void;
             onMouseOut(): void;
             onMouseOver(e?: PointerEventObject): void;
-            select(selected?: boolean, accumulate?: boolean): void;
+            select(selected?: boolean | null, accumulate?: boolean): void;
```

Comment:
Why select can be `null`?

Reply:
According to the API:
 `When true, the point is selected. When false, the point is unselected. When null or undefined, the selection state is toggled.` 
https://api.highcharts.com/class-reference/Highcharts.Point#select